types/reflection/serverreflection.go

```diff
@@ -34,7 +34,7 @@ To register server reflection on a gRPC server:
 
 	s.Serve(lis)
 */
-package gogoreflection // import "google.golang.org/grpc/reflection"
+package reflection // import "google.golang.org/grpc/reflection"
```

Comment:
I moved Frojdi's gogoproto registration fixes from `server/grpc` to `types/reflection`.

I think those fixes are needed in baseapp, because we call `reflection.GetFileDescriptorSet()` which reads from the gogoproto (fixed) registry. It seems cleaner to let baseapp import from `types` than from `server`